_posts/algorithm-development/guides/2016-10-13-git-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title:  "Using Git with Algorithmia"
+excerpt: "Access Algorithmia from Git"
+date: 2016-10-13  09:50:46
+permalink: /algorithm-development/git-support/
+tags: [algo-guide-git]
+show_related: false
+author: steph_kim
+image:
+    teaser: /icons/Algorithmia_Basics.png
+---
+
+Every algorithm on Algorithmia has a dedicated git repository. This allows you to seamlessly integrate Algorithmia with your current workflows,
+such as your code editor, IDE, or deployment pipelines. 
+
+<img src="/images/post_images/legit/legit-demo.gif" alt="Demo for Git" class="screenshot">
+
+## Accessing a repository
+
+You can clone existing algorithms from the following endpoint:
+{% highlight bash %}
+    git clone https://git.algorithmia.com/git/:username/:algoname.git
+{% endhighlight %}
+
+Replace the `:username` and `:algoname` portions with the currect values, case sensitive.
+
+## Committing changes to an algorithm
+
+You can apply your changes and push them as usual.
+{% highlight bash %}
+git commit -a -m "Note on changes goes here"
+git push
+{% endhighlight %}
+
+Everything will behave as a normal git repository, with the additional last line that will point you to a private REST API endpoint.
+
+{% highlight bash %}
+Remote: build successful for algo://:username/:algoname/:hash
+{% endhighlight %}
+
+The REST API endpoint is a live version of the latest build of your algorithm (previous versions are still available). It behaves exactly as a published algorithm 
+except for the fact that it is not listed in the marketplace and will only accept calls from your API keys. This is a great way 
```

Comment:
This is a bunch of detail, maybe instead say that this is the same as when you click `compile` in the Algorithmia web editor.